### DISCUSSION TITLE
(Not an issue) Is tunneling not available in ngrok free version

### DISCUSSION THREAD

**Discussion by satvikreddy** [voxos-ai/bolna#226] [Q&A] [answered]
This is NOT an issue.
I could not find 'discussions' section in this repo, so asking the community here.

After doing the local setup, I created a new ngrok account and provided authtoken.
However, I'm getting 'Upgrade your account' error.

Is this expected? Or did I miss some setup step.

**Comment by prateeksachan** [comment, marked as answer]
hey @satvikreddy so, ngrok has a limitation for the number of tunnels per agent for free users. In our example ngrok-config we've added 3 tunnels of which 2 are telephony servers. Ideally, there would be only 1 telephony server you'll need (either twilio or plivo). So the other one you can remove from the config.

Let me know if this works.

**Reply by satvikreddy** [reply]
Thanks, working now!!
